pvlib/tests/test_spectrum.py

```diff
@@ -315,3 +315,44 @@ def test_spectral_factor_caballero_supplied_ambiguous():
     with pytest.raises(ValueError):
         spectrum.spectral_factor_caballero(1, 1, 1, module_type=None,
                                            coefficients=None)
+
+
+@pytest.mark.parametrize("module_type,expected", [
+    ('asi', np.array([1.15534029, 1.1123772, 1.08286684])),
+    ('fs-2', np.array([1.0694323, 1.04948777, 1.03556288])),
+    ('fs-4', np.array([1.05234725, 1.037771, 1.0275516])),
+    ('multisi', np.array([1.03310403, 1.02391703, 1.01744833])),
+    ('monosi', np.array([1.03225083, 1.02335353, 1.01708734])),
+    ('cigs', np.array([1.01475834, 1.01143927, 1.00909094])),
```

Comment:
Can we include a test where the inputs and outputs are pandas Series?

You could re-use the same expected value variables, just convert them to Series and re-run the function like this example: https://github.com/pvlib/pvlib-python/blob/main/pvlib/tests/test_iam.py#L60-L63